### ISSUE TITLE
Addon incompatible Thunderbird 68

### ISSUE THREAD

**Issue by SecT0uch** [inverse-inc/sogo-integrator#1]
Would it be possible to update the addon [to make it compatible with Thunderbird 68+](https://developer.thunderbird.net/add-ons/updates) ?

**Comment by extrafu** [comment]
Yes we're working on it. But it's set to be merged into one extension (SOGo Connector + Integrator).

**Comment by SecT0uch** [comment]
Ok, I'll keep a track on this. Thanks
